### PR TITLE
OpenFST 1.8.3 compatibility

### PR DESCRIPTION
**Pull request by mmcauliffe**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
-{% set version = "5.5.1068" %}
+{% set version = "5.5.112" %}
```
Comment (h-vetinari) [inline recipe/meta.yaml:1]: How can we go back on the patch version here? Should the minor version have been bumped?

**Review by h-vetinari — COMMENTED** [review]
I think the version here is broken, it should be higher than the previous builds.

Also, out of curiosity, any news on upstreaming these patches into kaldi?

File: recipe/meta.yaml (hunk 1 above)
Comment (mmcauliffe) [reply recipe/meta.yaml:1]: Oops, fixed here: https://github.com/conda-forge/kaldi-feedstock/pull/45, sorry about that!

**Comment by mmcauliffe** [comment]
As for upstreaming the patches, there hasn't been any movement on the PRs for 1.8.X, and in general Kaldi is not being very actively maintained/developed since the devs are focused on new toolkits like K2 (https://github.com/k2-fsa), so I'm not optimistic that they'll ever be merged in.

**Pull request merged**